packages/frontend/tests/acceptance/chat/privately-with/-acceptance-test.ts

```diff
@@ -22,9 +22,34 @@ module('Acceptance | Chat | Privately With', function(hooks) {
   setupApplicationTest(hooks);
   clearLocalStorage(hooks);
 
+  module('is not logged in', function(hooks) {
+    hooks.beforeEach(async function() {
+      QUnit.config.altertitle = false;
```

Comment:
I was looking at the documentation for this, and it looks like it would need to go in test-setup.js